### PR TITLE
[Merged by Bors] - Enable wgpu device limits

### PR DESCRIPTION
**Pull request by Neo-Zhixing**
Follow up on #547 and #1401

### CODE REVIEW

File: crates/bevy_wgpu/src/lib.rs [hunk 1]
```diff
@@ -119,10 +108,12 @@ pub fn get_wgpu_render_system(resources: &mut Resources) -> impl FnMut(&mut Worl
 }
 
 #[derive(Default, Clone)]
-pub struct WgpuOptions {
+pub struct WgpuOptions<'a> {
+    pub name: Option<&'a str>,
```
Comment (cart) [inline crates/bevy_wgpu/src/lib.rs:112]: Can we make `name` a `Cow<'static, str>`? I don't want  "plugin configuration" types to have lifetimes.

Comment (Neo-Zhixing) [reply crates/bevy_wgpu/src/lib.rs]: Done!

**Comment by cart** [comment]
Haha sorry one more thing. Can we change `name` to `device_label`? WgpuOptions::name doesn't feel descriptive enough.

**Pull request closed without merging**